internal/worker.go

```diff
@@ -271,6 +271,12 @@ type (
 		//
 		// Deprecated: All bugports are always deprecated and may be removed at any time.
 		WorkerBugPorts WorkerBugPorts
+
+		// Optional: EventMonitoring provides a set of methods that can be used to collect
+		// stats on the Worker for debugging purposes.
+		// default: noop implementation
+		// Deprecated: in development and very likely to change
```

Comment:
nit: Deprecated is not the right word, but go does not have Unstable/Experimental annotation. 
I guess this should be enough to show IDEs to not rely on this.

Reply:
Yeah deprecated was the closest I found to a "do not use" annotation :)